avm/src/lib.rs

```diff
@@ -50,27 +49,21 @@ pub fn use_version(version: &Version) -> Result<()> {
     let installed_versions = read_installed_versions();
     // Make sure the requested version is installed
     if !installed_versions.contains(version) {
-        let input: String = Input::new()
-            .with_prompt(format!(
-                "anchor-cli {} is not installed, would you like to install it? (y/n)",
-                version
-            ))
-            .default("n".into())
-            .interact_text()?;
-        if matches!(input.as_str(), "y" | "yy" | "Y" | "yes" | "Yes") {
-            install_version(version)?;
-        } else {
-            println!(
-                "Version {} is not installed, staying on version {}.",
-                version,
-                current_version()?
-            );
-            return Ok(());
-        }
+        println!(
+            "Version {} is not installed, staying on version {}.",
+            version,
+            current_version()?
+        );
+
+        return Err(anyhow!(
+            "You need to run 'avm install {}' to install it before using it.",
+            version
+        ));
     }
 
     let mut current_version_file = fs::File::create(current_version_file_path().as_path())?;
     current_version_file.write_all(version.to_string().as_bytes())?;
+    println!("Now using avm Version {}.", current_version()?);
```

Comment:
```suggestion
    println!("Now using version {}.", current_version()?);
```

Reply:
oh man, i really shouldn't code while sick. I like the idea of telling the user what was installed

```
root@8d4fc19e1f61:/app# nvm use stable
Now using node v16.14.0 (npm v8.3.1)
root@8d4fc19e1f61:/app# 
root@8d4fc19e1f61:/app# 
root@8d4fc19e1f61:/app# nvm use v16.14.0
Now using node v16.14.0 (npm v8.3.1)
```

would it be reasonable to say `"Now using Anchor version {}"`?

Reply:
```
sven@x1carbon:~/src/cryptoworkbench/anchor/avm$ cargo run --bin avm use 0.22.0
   Compiling avm v0.22.1 (/home/sven/src/cryptoworkbench/anchor/avm)
    Finished dev [unoptimized + debuginfo] target(s) in 3.37s
     Running `/home/sven/src/cryptoworkbench/anchor/target/debug/avm use 0.22.0`
Now using anchor version 0.22.0.
sven@x1carbon:~/src/cryptoworkbench/anchor/avm$ cargo run --bin avm use 0.22.1
    Finished dev [unoptimized + debuginfo] target(s) in 0.14s
     Running `/home/sven/src/cryptoworkbench/anchor/target/debug/avm use 0.22.1`
Now using anchor version 0.22.1.
sven@x1carbon:~/src/cryptoworkbench/anchor/avm$ cargo run --bin avm use 0.21.0
    Finished dev [unoptimized + debuginfo] target(s) in 0.09s
     Running `/home/sven/src/cryptoworkbench/anchor/target/debug/avm use 0.21.0`
Version 0.21.0 is not installed, staying on version 0.22.1.
Error: You need to run 'avm install 0.21.0' to install it before using it.
sven@x1carbon:~/src/cryptoworkbench/anchor/avm$ echo $?
1
```